### PR TITLE
Use Clown

### PR DESCRIPTION
**Pull request by wholesomedev**

### CODE REVIEW

**Comment by forabi** [comment]
Aside from the snapshot date issue, LGTM!

**Review by forabi — APPROVED** [review]

**Comment by wholesomedev** [comment]
Will wait for the snapshot issue to be fixed before I merge this

**Comment by wholesomedev** [comment]
@forabi This is ready.

Please feel free to merge it.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "repository": "https://github.com/hollowverse/hollowverse.git",
   "scripts": {
-    "pretest": "npm-run-all --aggregate-output -s app/graphql/build -p -c validate-filenames lint",
```
Comment (forabi) [inline package.json:8]: `npm-run-all` can run commands in serial (`-s`) and parallel (`-p`). The order of `-s` and `-p` matters. We need the GraphQL types in order to correctly check TS code. I'll rewrite this command.

**Pull request merged**